### ISSUE TITLE
tempView query argument doesn't work.

### ISSUE THREAD

**Issue by warpr** [dominicbarnes/node-couchdb-api#17]
There are two related issues here:

First, db.tempView expects the query argument in args[2] instead of args[1].

Second, the query argument is not automatically encoded as a querystring, so it has to be passed as

```
var query = { "qs": { "key": JSON.stringify (["foo", "bar"]) } }
```

instead of:

```
var query = { "key": ["foo", "bar"] }
```

Some example code which illustrates this issue:  https://gist.github.com/4518125

**Comment by dominicbarnes** [comment]
Fixed (with test cases in 7dd7666bb27a0c5d6f97b1cc04fc4fa98a039a15) ... Please update to v1.2.2 and let me know if it works for you! :)

**Comment by warpr** [comment]
Yes, tempView works as expected now, thanks for fixing this so fast :)
